### PR TITLE
admin list nodes : doc update

### PR DESCRIPTION
**Pull request by vsian**
### What problem does this PR solve?

Update the document for cluster HTTP API list nodes.

### Type of change

- [x] Documentation Update

### CODE REVIEW

File: docs/references/http_api_reference.mdx [hunk 1]
```diff
@@ -3809,19 +3809,37 @@ The response includes a JSON object like the following:
...
   `0`: The operation succeeds.
-- `"nodes" : array` : 
-  Each element is in `[nodename, noderole]` format.
+- `"nodes" : array` :
+  Each element is an object with following information:
+    - `"address"` : the address of the node.
+    - `"heartbeat"` : only valid for non-leader role, this is the number of heartbeat message received from the leader.
+    - `"name"` : the node's name.
+    - `"role"` : the node's role.
+    - `"status"` : the current status of the node.
```
Comment (writinwaters) [inline docs/references/http_api_reference.mdx:3842]:
```suggestion
    - `"status"` : The node's status.
```

Comment (writinwaters) [reply docs/references/http_api_reference.mdx:3842]: Please also list available options. offline?

Comment (writinwaters) [reply docs/references/http_api_reference.mdx:3842]: missing `"last_update"`.

**Review by writinwaters — CHANGES_REQUESTED** [review]

**Review by writinwaters — APPROVED** [review]
LGTM.

**Review by JinHai-CN — APPROVED** [review]

**Pull request merged**